plugins/announcements-velocity/src/main/java/net/civmc/announcements/ProxyListener.java

```diff
@@ -0,0 +1,5 @@
+package net.civmc.announcements;
+
+public class ProxyListener {
```

Comment:
what does this do?

Reply:
guess i forgot to remove this class, is the recommended place for event listeners on the proxy though iirc. Will need this for the future changes i want, but forget which ones need it.